### ISSUE TITLE
Change github default branch to 1.x

### ISSUE THREAD

**Issue by stevector** [pantheon-systems/terminus-secrets-plugin#9]
Perhaps this should wait until the 1.0 launch?

**Comment by greg-1-anderson** [comment]
Hard to say when this should happen. Switching the default branch will break any CI that is using the   0.x version of this plugin via `git clone` without a branch selection. I don't know if anyone is doing this.

Having to switch to the 1.x branch is a minor inconvenience. At some point, we should flip it over. Maybe a bit after Terminus 1.0.0-stable.

**Comment by stevector** [comment]
Good point :) I'm doing it here: https://github.com/stevector/nerdologues-d8/blob/master/scripts/circle-ci/deploy-to-multidev.sh#L21

I think waiting until 1.0.0-stable is fine.

**Comment by NicholasRBowers** [comment]
Speaking as a new Pantheon user, I definitely skipped to the Installation instructions on the default `0.x` branch and everything failed. Took me awhile to realize that the default Terminus version available for download is `1.x` and I needed to switch branches.  Might be about time we make this change. Unsure how to prevent legacy CircleCI builds from breaking.

**Comment by greg-1-anderson** [comment]
I resolved this by combining the 0.x and 1.x versions of this plugin into a single branch. I don't recommend replicating this pattern, save for instances where there are existing CI scripts using `git clone` to install the old version via the default branch.
